### ISSUE TITLE
List of habitats for section-level details

### ISSUE THREAD

**Issue by xaviermestdagh** [BiologicalRecordsCentre/eBMS#42]
Here attached a first selection of EUNIS habitats ("eBMS"=1), with four levels.

1. to be checked by partners from other european regions?

2. to be checked by partners from non-european regions to complete this EUNIS selection? Does France have a list of habitats from their DOM-TOM?  

[Section_habitat_details_sorted_20170626_xm.xlsx](https://github.com/BiologicalRecordsCentre/eBMS/files/1102217/Section_habitat_details_sorted_20170626_xm.xlsx)

**Comment by DavidRoy** [comment]
For the UKBMS we reduced the set of EUNIS categories to those relevant to butterflies.  It may be worth considering this for eBMS - i.e. do we need the marine categories for butterfly monitoring?

[UKBMS G4 Transect habitat classification.pdf](https://github.com/BiologicalRecordsCentre/eBMS/files/1102316/UKBMS.G4.Transect.habitat.classification.pdf)

**Comment by xaviermestdagh** [comment]
Yes you are right. I let all EUNIS habitats in the attached table, but only those with field eBMS=1 were considered pertinent for butterflies (191 habitats).

**Comment by DavidRoy** [comment]
That works.  I'd missed that column
